inspire/ext/formatter_jinja_filters/record.py

```diff
@@ -262,6 +263,11 @@ def link_to_hep_affiliation(record):
                 str(record['department_acronym'])
 
 
+def join_nested_lists(l):
+    new_list = [item for sublist in l for item in sublist]
+    return ', '.join(element for element in new_list)
```

Comment:
Yeah, that also works! I amended the commit. ;)

2015-10-27 8:39 GMT+01:00 Javier Martin Montull <notifications@github.com>:

> In inspire/ext/formatter_jinja_filters/record.py
> <https://github.com/inspirehep/inspire-next/pull/440#discussion_r43090896>
> :
>
> > @@ -262,6 +263,11 @@ def link_to_hep_affiliation(record):
> >                  str(record['department_acronym'])
> >
> >
> > +def join_nested_lists(l):
> > +    new_list = [item for sublist in l for item in sublist]
> > +    return ', '.join(element for element in new_list)
>
> so new_list is just a list of elements?
>
> ', '.join(new_list)
>
> does not work?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/inspirehep/inspire-next/pull/440/files#r43090896>.
>